src/components/ToggleChip.tsx

```diff
@@ -10,40 +12,53 @@ export interface ToggleChipProps<X> {
   onClick: () => void;
   xss?: X;
   disabled?: boolean;
+  icon?: IconKey;
+  clearable?: boolean;
```

Comment:
@ksk5280 just curious, but per "Adds a clearable prop to make the X icon optional" --> is `clearable=false` basically then "a regular `Chip`"?

Maybe add a `/** ...description of use case... */` for when users would want a non-clearable ToggleChip.

Reply:
good point, the design shows chips that are without the x but still have a hover and active state, but I'm not sure what those would be used for. I'll ask Andrea about this